modules/logging-collector-alerts.adoc

```diff
@@ -1,12 +1,12 @@
 // Module included in the following assemblies:
 //
-// * logging/logging_alerts/default-logging-alerts.adoc
+// * logging_alerts/default-logging-alerts.adoc
 
 :_content-type: REFERENCE
 [id="logging-collector-alerts_{context}"]
 = Logging collector alerts
 
-In logging 5.8 and later versions, the following alerts are generated by the {clo}. You can view these alerts in the {ocp-product-title} web console.
+The following alerts are generated by the {clo}. You can view these alerts in the {ocp-product-title} web console.
```

Comment:
There isn't CollectorHighErrorRate CollectorVeryHighErrorRate, the replacement is CollectorHigh403ForbiddenResponseRate.  https://github.com/openshift/runbooks/blob/master/alerts/cluster-logging-operator/CollectorHigh403ForbiddenResponseRate.md

And there is a new alert DiskBufferUsage.
Detected consuming too much node disk on $labels.hostname host